README.md

```diff
@@ -25,6 +25,10 @@ To learn more about the Online DevOps Dojo and the Pet Clinic multi-verse in whi
 
 * The pull request / feature branch version control model.
 
+* The team deciding in an incident room strewn with coke cans, pizza boxes and stale air t to investigate the use of value stream mapping to better understand their current and desired processes.
```

Comment:
typo (stale air t to) , and sentence may need a bit of rewording?

Reply:
done